src/util/filter/fliter.notFoundException.ts

```diff
@@ -0,0 +1,22 @@
+import { ArgumentsHost, Catch, HttpStatus } from '@nestjs/common';
+import { BaseExceptionFilter } from '@nestjs/core';
+import { Prisma } from '@prisma/client';
+import { Response } from 'express';
+
+@Catch(Prisma.PrismaClientKnownRequestError)
+export class PrismaClientExceptionFilter extends BaseExceptionFilter {
+  catch(exception: Prisma.PrismaClientKnownRequestError, host: ArgumentsHost) {
+    const ctx = host.switchToHttp();
+    const response = ctx.getResponse<Response>();
+
+    if (exception.code == 'P2025') {
```

Comment:
Explain what p2025 means? In this case I would create a one of enum that has the keys named properly and has the specific values associated with it, e.g. 
```typescript
export enum DatabaseExceptionsEnum {
  notFound = 'P2025'
}
```